libs/cardano-ledger-core/test/Test/Cardano/Ledger/BaseTypesSpec.hs

```diff
@@ -17,7 +17,6 @@ import Data.Typeable
 import Test.Cardano.Ledger.Binary.RoundTrip (roundTripCborExpectation)
 import Test.Cardano.Ledger.Common
 import Test.Cardano.Ledger.Core.Arbitrary ()
-import Test.Cardano.Ledger.Core.Utils (epsilonMaybeEq)
```

Comment:
`epsilonMaybeEq` was only used here, so we might want to remove the function itself as well. Unless we think that it might be useful again at some point.

Reply:
I think we can remove it. If we ever need it again, we can always bring it back. I have a variant of it in a couple of projects of mine.
I doubt though that we will need it, since it is usually only useful for testing floating point numbers and other computation related to rounding, which is unlikely to be useful in ledger again.